cl/alerts/tasks.py

```diff
@@ -133,6 +137,33 @@ def get_docket_alert_recipients(
     )
 
 
+def get_docket_notes_and_tags_by_user(
+    d_pk: int, user_pk: int
+) -> tuple[str | None, list[UserTag]]:
+    """Get user notes and tags for a docket.
+
+    :param d_pk: Docket primary key
+    :param user_pk: The User primary key
+    :return: A two tuple of docket notes or None if not available, a list of
+    tags assigned to the docket.
+    """
+
+    notes = None
+    user_tags = []
+    favorite = (
+        Favorite.objects.filter(docket_id=d_pk, user_id=user_pk)
+        .only("notes")
+        .first()
+    )
+    if favorite and favorite.notes:
+        notes = favorite.notes
+
+    tags = UserTag.objects.filter(user_id=user_pk, dockets__id=d_pk)
+    for tag in tags:
+        user_tags.append(tag)
```

Comment:
Why do we need this?

Reply:
yeah, this is not necessary, now I just converted the `QuerySet` to a `list()` since it's simpler to compare in the test.
Thanks!